bundles/net.tourbook/src/net/tourbook/ui/views/nutrition/DialogSearchProduct.java

```diff
@@ -0,0 +1,853 @@
+/*******************************************************************************
```

Comment:
When a product is not found, then the search field should get the focus and the content should be selected

![image](https://github.com/mytourbook/mytourbook/assets/1283445/dda4c74c-f089-413b-85d8-3f9fe4dc50f2)

Reply:
Good idea, fixed